components/swatch/index.css

```diff
@@ -10,38 +10,86 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-.🤫 {
-  /* hack: make sure we don't complain about this color being undefined */
-  --spectrum-picked-color: 0;
+/* Swatch tokens */
+.spectrum-Swatch {
+  /* Static properties - no tokens */
+  --spectrum-swatch-focus-ring-border-radius: 8px;
+
+  --spectrum-swatch-border-color-regular: rgba(0, 0, 0, 0.5);
+  --spectrum-swatch-border-color-light: rgba(0, 0, 0, 0.2);
+
+  /* Size */
+  --spectrum-swatch-size: var(--spectrum-swatch-size-small);
+  --spectrum-swatch-border-radius: var(--spectrum-corner-radius-100);
+  --spectrum-swatch-border-thickness: var(--spectrum-border-width-100);
+  --spectrum-swatch-border-thickness-selected: var(--spectrum-border-width-200);
+  --spectrum-swatch-disabled-icon-size: var(--spectrum-workflow-icon-size-75);
+  --spectrum-swatch-slash-thickness: var(--spectrum-swatch-slash-thickness-small);
+  --spectrum-swatch-focus-ring-thickness: var(--spectrum-focus-ring-thickness);
+  --spectrum-swatch-spacing-selected: var(--spectrum-swatch-border-thickness-selected) * 4;
```

Comment:
My yarn run dev is failing because of the unused custom property on line 29.

Reply:
Sorry about that. I had to restart my build to catch this.